book/website/reproducible-research/open/open-hardware.md

```diff
@@ -1,111 +1,359 @@
+If they wish, they can proceed to share such modifications.
+
+
+
+Although OSH projects are diverse in their degree of "openness", best practices in OSH guide us to identify a core, minimum documentation that needs to be in place so others can study, modify, distribute, make or sell our hardware.
+We will go through these various components in the following sections. 
+
+
+## Types and phases of open source hardware 
+
+```{figure} ../../figures/open-hardware.*
+---
+height: 500px
+name: open-hardware
+alt: 'Illustration showing the life cycle of an open hardware project in research, using seeds and gardening tools as an analogy. The figure shows a circle with six stages surrounding the open hardware logo and gardening tools (a fork, shears and watering can), going clockwise starting on the top. Stage 1 NEED: it represents a researcher thinking about a seed that represent an idea that would solve its problem. Stage 2: Prototyping: Three different seeds are represented, the last one has flowers and roots, which are labeled. This represents the action to try out or "prototyping" different tools to address the need. Stage 3: Selection/Demo: Next to gardening tools, the third prototype is represented as a schematic on a board and a researcher is writing notes. It represents a stage where a prototype can be selected for further development. Stage 4: Documentation: Several boards present separately the roots, the flower and a fork, Six researchers are writing notes about the boards. This represent the inclusion of, these design choices and criteria in the project documentation, so others can follow and learn from the design process. Stage 5: Publishing: An actor is throwing seeds out of a window, another person is receiving the seeds with open arms; on the top we can see three square with labels on it: Reuse, Replicate, Reproduce. This represents the moment a researcher shares openly the designs and documentation of the new device, which can be reused, replicated or reproduce by others to better understand how data was produced. Stage 6: Cultivate ecosystem of open knowledge: A watering can pour water on five plants that are in the soil, two people can be seen on the other side looking at the plants. This represents the hardware commons, the new piece of knowledge can become a building block for others to explore new research questions or scenarios, especially when work is invested in growing a community.'
+---
+_The Turing Way_ project illustration by Scriberia. Used under a CC-BY 4.0 licence. DOI: [10.5281/zenodo.3332807](https://doi.org/10.5281/zenodo.3332807).
+```
+
+
+
+
+OSH projects emerge as **a response to a concrete need**. 
+For example, you may need to process lots of samples, or to measure a parameter you were not considering before, or to take measurements outside of the lab. 
+With the analyzing of your need, your OSH project starts.
+It is advisable to scope the project at this point (see project scope chapter, In preparation).
```

Comment:
is there mean to be a link here to a Project Scoping chapter?

Also is the "in preparation" meant to be here?